### PR TITLE
AO3-3463 Move time_in_zone out of comment cache blocks

### PR DESCRIPTION
**Pull request by EchoEkhi**
# Pull Request Checklist

* [x] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [x] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [x] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [x] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [x] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-3463

## Purpose

Fix inconsistent time zones in comments.

## Testing Instructions

See JIRA ticket.

## Credit

EchoEkhi (He/him)

### CODE REVIEW

File: app/views/comments/_single_comment.html.erb [hunk 1]
```diff
@@ -54,12 +56,12 @@
...
         <% end %>
         <blockquote class="userstuff"><%=raw sanitize_field(single_comment, :comment_content) %></blockquote>
-        <% unless single_comment.edited_at.blank? %>
-          <p class="edited datetime">
-            <%= ts("Last Edited") %> <%= time_in_zone(single_comment.edited_at) %>
-          </p>
-        <% end %>
-      <% end %><!-- end caching -->
+      <% end %>
+      <% unless single_comment.edited_at.blank? %>
```
Comment (weeklies) [inline app/views/comments/_single_comment.html.erb:60]: We could improve the expression to be more direct here `if single_comment.edited_at.present?`.

**Review by weeklies — COMMENTED** [review]
It would be good to add a test [here](https://github.com/otwcode/otwarchive/blob/72caa938719e30f6fd273b7408b0935a3f762a19/features/comments_and_kudos/add_comment.feature).

File: app/views/comments/_single_comment.html.erb [hunk 2]
```diff
@@ -14,9 +14,9 @@
     <% elsif !can_see_hidden_comment?(single_comment) %>
       <p class="message"><%= ts("(This comment is under review by an admin and is currently unavailable.)") %></p>
     <% else %>
-      <% cache([single_comment, single_comment.comment_owner], expires_in: 1.week) do %>
-        <% # FRONT END, update this if a.user comes in %>
-        <h4 class="heading byline">
+      <%# FRONT END, update this if a.user comes in %>
```
Comment (ceithir) [inline app/views/comments/_single_comment.html.erb:17]: Is there any value in keeping around this 12 years old comment?

**Review by ceithir — APPROVED** [review]

File: app/views/comments/_single_comment.html.erb (hunk 2 above)
Comment (EchoEkhi) [reply app/views/comments/_single_comment.html.erb:17]: I don't even know what it means so I decided to leave it 🤷‍♂️

**Pull request merged**